File: skeleton/src/main/java/me/shkschneider/skeleton/SkeletonFragmentActivity.java
package me.shkschneider.skeleton;

import android.support.annotation.NonNull;
import android.support.v4.app.Fragment;

public class SkeletonFragmentActivity extends SkeletonActivity {

    protected void setContentFragment(@NonNull final Fragment fragment) {
        getSupportFragmentManager().beginTransaction()
                .replace(R.id.frameLayout, fragment)
                .commit();
    }

    @Override
    protected void onResumeFragments() {
        super.onResumeFragments();
    }

}

File: core/src/main/kotlin/me/shkschneider/skeleton/SkeletonFragmentActivity.kt
package me.shkschneider.skeleton

import android.os.Bundle
import androidx.fragment.app.Fragment

/**
 * @see SkeletonActivity
 */
abstract class SkeletonFragmentActivity : SkeletonActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.sk_fragmentactivity)
    }

    protected fun setContentFragment(fragment: Fragment) {
        supportFragmentManager.beginTransaction()
                .replace(R.id.frameLayout, fragment)
                .commit()
    }

    override fun onResumeFragments() {
        super.onResumeFragments()
        // fragmentManager.executePendingTransactions()
    }

    override fun onSupportNavigateUp(): Boolean {
        return super.onSupportNavigateUp()
    }

    override fun onBackPressed() {
        if (supportFragmentManager?.backStackEntryCount != 0) {
            supportFragmentManager.popBackStack()
            return
        }
        super.onBackPressed()
    }

}
